.github/workflows/test-python.yml

```diff
@@ -56,19 +56,18 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install a specific version of uv
-        uses: astral-sh/setup-uv@v3
+        uses: astral-sh/setup-uv@v6
         with:
-          enable-cache: true
-          version: "0.5.x"
+          version: "latest"
 
       - name: uv sync
         working-directory: python
         run: uv sync --no-install-package geoindex-rs
```

Comment:
Actually, it should be possible to just remove the `uv sync ...` part, and just do `uv run maturin develop && uv run pytest` below?